pkg/constants/constants.go

```diff
@@ -16,9 +16,13 @@ var (
 
 // KFService Constants
 var (
-	KFServiceName        = "kfservice"
-	KFServiceAPIName     = "kfservices"
-	KFServicePodLabelKey = KFServingAPIGroupName + "/" + KFServiceName
+	KFServiceName                       = "kfservice"
+	KFServiceAPIName                    = "kfservices"
+	KFServicePodLabelKey                = KFServingAPIGroupName + "/" + KFServiceName
+	KFServiceS3SecretEndpointAnnotation = KFServingAPIGroupName + "/" + "s3-endpoint"
+	KFServiceS3SecretRegionAnnotation   = KFServingAPIGroupName + "/" + "s3-region"
+	KFServiceS3SecretSSLAnnotation      = KFServingAPIGroupName + "/" + "s3-verifyssl"
```

Comment:
Should these live here or just in the file where they're used? I don't imagine we'll be using these in multiple places and I want to avoid the uncontrolled growth of this file.